### ISSUE TITLE
[Camera][iOS] Please restore ability for iPhones to record MPEG-4/H264 (not HEVC/H265)

### ISSUE THREAD

**Issue by cfchris** [flutter/flutter#89234]
### TLDR

* I'm fairly sure the camera plugin (on iOS) used to record MPEG-4/H264. 
* Now, some iOS devices are recording [HEVC/MPEG-H](https://en.wikipedia.org/wiki/High_Efficiency_Video_Coding). 
* Browsers and Android don't have wide support for HEVC. 
* Our app needs MPEG-4 back.

### Details

Up until recently (last month), we were running a fork of the "camera" plugin because of the [Android Audio issue](https://github.com/flutter/plugins/pull/3124). After a few months pause on our App, I resumed work. I updated Flutter and all packages including "camera" (because the audio issue was finally fixed).

With the current version of the "camera" plugin (0.9.2+1) some (but not all) of our iOS users are producing videos in HEVC format.

![image](https://user-images.githubusercontent.com/819847/131547300-84941841-1542-476c-b864-4d80e1d9fcf6.png)

Those videos are part of the "texting" tools in our CRM. When the videos are HEVC, they show up as an audio player on Android and in most browsers.

![image](https://user-images.githubusercontent.com/819847/131547812-bad68dfd-4c54-40d3-8290-e805c6876f9f.png)

Until the HEVC(MPEG-H) codec is more widely supported, I would like our mobile app to produce standard MPEG-4 files.

Can we have a setting to controller this? Or is this a regression that should be "fixed"?

(FWIW: The [iOS code for the camera plugin](https://github.com/flutter/plugins/blob/1f502e8b6f0082a0f2e2e89e223b7f2de899177e/packages/camera/camera/ios/Classes/CameraPlugin.m#L819) creates a file with extension ".mp4". That file should contain video in the "MPEG-4" format, not "MPEG-H/HEVC".)

**Comment by cfchris** [comment]
I don't know why this affects some phones and not others. 

The (Settings > Camera > Formats) settings should affect this. "High Efficiency" should be HVEC and "Most Compatible" should be MPEG-4.

I have an iPhone 7 Plus that is set to "High Efficiency" (the default) which seems like it would have the problem. But, it is producing videos in MPEG-4/H264.

My coworker has and iPhone SE 2nd generation and regardless of the camera formats setting, it is producing MPEG-H/HEVC/H265 video files.

**Comment by cfchris** [comment]
This could be the cause of https://github.com/flutter/flutter/issues/87911.

**Comment by TahaTesser** [comment]
Hi @cfchris 
I see there's an open issue addressing the case you described.
Please follow up on that issue, Closing as duplicate of https://github.com/flutter/flutter/issues/83074
If you disagree, please write in the comments and I will reopen it.
Thank you
